config/initializers/periodic_jobs.rb

```diff
@@ -95,6 +95,10 @@ def with_each_shard_by_database(klass, method, *args)
     with_each_shard_by_database(StreamItem, :destroy_stream_items_using_setting)
   end
 
+  Delayed::Periodic.cron "Hourly course touch", "0 1 * * *" do
```

Comment:
Should we do this more than hourly?

Reply:
proposal passed to set it to 15 mins.

Mark, Paul, Mike